include/BasicFilters.h

```diff
@@ -328,6 +328,16 @@ class BasicFilters
 		}
 	}
 
+	inline void setSampleRate(float sampleRate)
```

Comment:
```suggestion
	inline void setSampleRate(sample_rate_t sampleRate)
```

Reply:
Yes, but `sample_rate_t` is an alias for `uint32_t` not `float`. So a cast to make that explicit would probably be better.

OR, since it seems there are only 3 usages of `sample_rate_t` in BasicFilters.h and in each case they are just converted to `float` anyway, maybe the constructor parameter and this method's parameter could both use `float`? I haven't checked the calling code yet to see what kind of effect that would have, but it would eliminate unnecessary casting between `float` and `uint32_t` in this class.